README.md

```diff
@@ -43,3 +43,8 @@ The application will now be available wherever you access it
 - Josh Lewis - [@josh-lew](github.com/josh-lew)
 - Maria Fedorec-Kyte - [@mfk07](github.com/mfk07)
 - Mikey Ying - [@mikeycodingstuff](github.com/mikeycodingstuff)
+
+### Acknowledgements
+A big thank you to Emily Clarkson, who provided all of the artwork for the project and tirelessly worked to ensure that its best.
```

Comment:
I reckon remove the 'and tirelessly worked to ensure that its best.'

Reply:
Done